src/users/__tests__/userSelectors-test.js

```diff
@@ -208,3 +209,28 @@ describe('getUsersById', () => {
     expect(result).toEqual(expectedResult);
   });
 });
+
+describe('getOtherUsers', () => {
+  test('returns all users but current user', () => {
+    const state = deepFreeze({
+      users: [
+        { id: 1, email: 'abc@example.com', isActive: true },
+        { id: 2, email: 'def@example.com', isActive: true },
+        { id: 3, email: 'xyz@example.com', isActive: true },
+      ],
+      accounts: [
+        { id: 1, email: 'abc@example.com', isActive: true },
```

Comment:
This is not the correct format for the account state. 
Also, include only one, as this is the minimal amount needed for the test.

Reply:
I think we only need email for checking this so all other fields could be safely removed.